src/components/ReportActionItemIOUPreview.js

```diff
@@ -0,0 +1,161 @@
+
+        // Email address of the creator of this iou report
+        ownerEmail: PropTypes.string,
+
+        // Outstanding amount of this transaction
+        cachedTotal: PropTypes.string,
+    }),
+
+    // All of the personal details for everyone
+    personalDetails: PropTypes.objectOf(personalDetailsPropTypes).isRequired,
+
+    // Session info for the currently logged in user.
+    session: PropTypes.shape({
+        // Currently logged in user email
+        email: PropTypes.string,
+    }).isRequired,
+};
+
+const defaultProps = {
+    iou: null,
+};
+
+class ReportActionItemIOUPreview extends React.Component {
+    /**
+     * Creates IOU preview pay button.
+     *
+     * @returns {View}
+     */
+    createPayButton() {
+        return (
+            <TouchableOpacity
+                style={[styles.buttonSmall, styles.normalRadius, styles.buttonSuccess, styles.mt6]}
+            >
+                <Text
+                    style={[
+                        styles.buttonSmallText,
+                        styles.buttonSuccessText,
+                    ]}
+                >
+                    Pay
+                </Text>
+            </TouchableOpacity>
+        );
+    }
+
+    /**
+     * Creates IOU preview box view if outstanding amount is not 0.
+     *
+     * @returns {View}
+     */
+    createPreviewBox() {
+        const managerName = lodashGet(
+            this.props.personalDetails,
+            [this.props.iou.managerEmail, 'displayName'],
+            this.props.iou.managerEmail,
+        );
+        const ownerName = lodashGet(
+            this.props.personalDetails,
+            [this.props.iou.ownerEmail, 'displayName'],
+            this.props.iou.ownerEmail,
+        );
+        const managerAvatar = lodashGet(this.props.personalDetails, [this.props.iou.managerEmail, 'avatar'], '');
+        const ownerAvatar = lodashGet(this.props.personalDetails, [this.props.iou.ownerEmail, 'avatar'], '');
+        const sessionEmail = lodashGet(this.props.session, 'email', null);
+        const cachedTotal = this.props.iou.cachedTotal.replace(/[()]/g, '');
+        const amount = Number(cachedTotal.substring(1));
+
+        if (amount === 0) {
+            return null;
+        }
+
+        return (
+            <View style={[styles.iouPreviewBox, {width: this.props.isSmallScreenWidth ? '100%' : '50%'}]}>
+                <View style={[styles.flexRow]}>
+                    <View style={styles.flex1}>
+                        <Text style={styles.h1}>{cachedTotal}</Text>
+                        <Text style={styles.mt2}>
+                            {managerName}
+                            {' owes '}
+                            {ownerName}
+                        </Text>
+                    </View>
+                    <MultipleAvatars
+                        avatarImageURLs={[managerAvatar, ownerAvatar]}
+                        styles={{
+                            secondAvatar: {
```

Comment:
is it possible to reuse the existing secondAvatar style? Or move this to a new styles object

Reply:
Existing secondAvatar style looks like this and I had to use custom style to put avatars in same line as in the issue description. Let me move this to a new styles object.

<img width="568" alt="Screenshot 2021-03-31 at 21 40 35" src="https://user-images.githubusercontent.com/3853003/113208552-d81e0700-9269-11eb-9c63-87ec489bfc70.png">